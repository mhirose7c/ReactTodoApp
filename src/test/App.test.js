import React from 'react';
import App from '../components/App';
import renderer from 'react-test-renderer';

it('matches to the snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot()
})