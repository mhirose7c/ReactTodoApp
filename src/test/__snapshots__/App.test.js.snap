// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches to the snapshot 1`] = `
<div>
  <div
    className="main-wrapper"
  >
    <div
      className="main"
    >
      <div
        className="copy-container"
      >
        <h1>
          Todo
        </h1>
        <h2>
          カテゴリー別にタスクを管理する
        </h2>
      </div>
      <div
        className="category-continer"
      >
        <div
          className="category-wrapper"
        >
          <ul>
            <li
              className="category-item"
            >
              <input
                onBlur={[Function]}
                onChange={[Function]}
                placeholder="カテゴリーの追加"
                value=""
              />
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
